Pipfile

```diff
@@ -4,9 +4,11 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
+paramiko = "*"
+pysftp = "*"
```

Comment:
@bobzoller you don't even need to worry about putting these here. The `Pipfile` is not really even in use for this library anymore, since we handle all dependencies through `setup.py` and `dev-requirements.txt`